hawkular-command-gateway-itest/pom.xml

```diff
@@ -32,11 +32,19 @@
     <debug.failsafe.argLine />
     <debug.wildfly.argLine />
     <version.com.squareup.okhttp>2.4.0</version.com.squareup.okhttp>
+    <version.com.samskivert.jmustache>1.11</version.com.samskivert.jmustache>
   </properties>
 
   <dependencies>
 
     <dependency>
+      <groupId>com.samskivert</groupId>
+      <artifactId>jmustache</artifactId>
```

Comment:
@pilhuhn It is used by several tests on agent to resolve a ```{{sessionId}}``` placeholder when the method ```expectText``` is used to test a response.

In order to remove this dependency we need to modify those tests.